packages/compass-components/src/compass-font-sizes.ts

```diff
@@ -1 +1,2 @@
+export const smallFontSize = '13px';
 export const defaultFontSize = '14px';
```

Comment:
Yes it exists there. We're not using this `defaultFontSize` in many places in Compass. I can open up a pr replacing those and removing it, but it's probably not worth doing here as its a bit unrelated. How do you feel about the addition of the `smallFontSize` ? Is there another way we might want to do this? We could add some typography components to avoid a constant here.

Reply:
This font size is completely out of leafygreen design system, right? Is the difference that noticeable that we can't use 14px here? If it is then I guess it's fine, but 1px sounds like something that can be changed to a slightly bigger font without major issues. Anyway, totally not something that should block this PR from landing

Reply:
I think so yes. Claudia mentioned the 13 px size explicitly but 14px I'm thinking could work, I'll check with her. In the meantime I think I'll merge and we can update those fonts independently (and remove the 14px).